examples/example-base/example-base.cpp

```diff
@@ -25,6 +25,10 @@ Slang::Result WindowedAppBase::initializeBase(const char* title, int width, int
     gWindow->events.sizeChanged = Slang::Action<>(this, &WindowedAppBase::windowSizeChanged);
 
     // Initialize the rendering layer.
+#ifdef _DEBUG
+    // Enable debug layer in debug config.
+    gfxEnableDebugLayer();
+#endif
```

Comment:
Whey can't the debug layer stuff be something passed in via `IDevice::Desc`? I'm confused why it needs to be global state.